app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,19 @@
+<div class = "<%= dom_id(sender)%>_<%=dom_id(recipient)%>_follow_unfollow">
```

Comment:
Good start. Could you find a way to make your div class selector more specific? Perhaps consider selecting the individual buttons.